python/ctsm/crop_calendars/regrid_ggcmi_shdates.py

```diff
@@ -32,47 +33,106 @@ def define_arguments(parser):
     parser.add_argument(
         "-rt",
         "--regrid-template-file",
-        help="Template file to be used in regridding of inputs.",
+        help="Template netCDF file to be used in regridding of inputs. This can be a CLM output file (i.e., something with 1-d lat and lon variables) or a CLM surface dataset (i.e., something with 2-d LATIXY and LONGXY variables).",
         type=str,
         required=True,
     )
     return parser
 
 
+# lat or lon
+def import_coord_1d(ds, coordName):
```

Comment:
I like putting this into a standalone function. This means you can easily unit test it, and
potentially use it elsewhere.

Are there other places that could use this function? If so it possibly could be moved to
a more generic place. Or it might be fine to leave it here, and you could just use this
this function from this module.

Reply:
I would like to see some unit tests added for this. There are some things that could be easily unit tested here. It's good to add unit tests for error checks, as they lend themselves to unit tests, but not to system tests usually.

There is also the system test that's run and that will test a lot of the code for correctness. So the unit tests could supplement those tests for parts of the code that aren't tested in the rxcropmaturity.py system test.

Reply:
- Moved to python/ctsm/utils.py in 51cc972.
- Unit testing added in 96b88aa and 04d2d73.